mhkit/dolfyn/rotate/api.py

```diff
@@ -0,0 +1,273 @@
+from . import vector as r_vec
+from . import awac as r_awac
+from . import signature as r_sig
+from . import rdi as r_rdi
+from .base import _make_model
+import numpy as np
+import xarray as xr
+import warnings
+
+
+# The 'rotation chain'
+rc = ['beam', 'inst', 'earth', 'principal']
+
+rot_module_dict = {
+    # Nortek instruments
+    'vector': r_vec,
+    'awac': r_awac,
+    'signature': r_sig,
+    'ad2cp': r_sig,
+
+    # TRDI instruments
+    'rdi': r_rdi}
+
+
+def rotate2(ds, out_frame='earth', inplace=False):
```

Comment:
Whats the reasoning behind naming this rotate2? Is it meant to be read like "rotate to...."? I think it may seem a bit confusing to others at first. Maybe we can consider changing it to something like "rotate_coordinates"?

Reply:
The first, yes. Dolfyn data was originally stored as an object, so the function was originally applied as "data.rotate2('earth')", but xarray doesn't support functions like classes do.